languages/go/internal/host/reflect.go

```diff
@@ -5,6 +5,16 @@ import (
 	"reflect"
 )
 
+// Checking for type equality is currently done with exact matching
+// We will not check if Go would permit a conversion between types,
+// or if the instance is an instance of a newtype, as doing so allows for type
```

Comment:
Is this conjunct correct? I thought we _do_ check "if the instance is an instance of a newtype" — we just don't check if the instance is an instance of the underlying wrapped type?

Reply:
Your understanding is correct, the comment is wrong